wordle-web/pages/about.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <v-container>
+    <v-card
+      color="secondary accent-4 text-primary"
+      class="mx-auto mt-8 w-75 pa-8 rounded text-center"
+      elevation="12"
+    >
+      <v-card-text class="text-center">
+        <v-card-title class="font-weight-bold" style="color: #FFFFFF; font-size:24px;">
+          About LexiQuest
+        </v-card-title>
+        <v-card-text class="font-italic" style="color: #FFFFFF; font-size: 20px;">
+          LexiQuest is an enchanting venture into the world of words, designed to combine the challenge of word puzzles with the allure of fantasy themes. Created as a class project, LexiQuest serves as a portal to mastering Vue and Vuetify technologies while engaging in a spellbinding experience. Here, each puzzle is not just a game, but a step towards unraveling the secrets of software craftsmanship in a realm where magic meets logic.
```

Comment:
Missing meg :(
What if she was an ancient wizard that inspired this :0

Reply:
Love this idea!